chapters/modules/chapter_1/pages/deploying_contracts.adoc

```diff
@@ -2,6 +2,204 @@
+
+[,Bash]
+----
+#cd into your .cairo folder
+cd ~/.cairo
+
+#Create the hello_starknet.cairo file in your .cairo folder and compile it
+cargo run --bin starknet-compile -- hello_starknet.cairo hello_starknet.json --replace-ids
+
+----
+
+Now that you have your account ready from Step 3, let’s declare your compiled contract and deploy it using the new account.
+
+[,Bash]
+----
+starknet declare --contract hello_starknet.json --account test_v11
+----
+
+____
+NOTE: If the declare command throws the following error: Error: OSError: [Errno 8] Exec format error: '/Users/gyan/cairo_venv/lib/python3.9/site-packages/starkware/starknet/compiler/v1/bin/starknet-sierra-compile' - then, you need to copy the starknet-sierra-compile from the .cairo folder to add it to the Python Libs using the following command:
+[,Bash]
+----
+#Assuming that you are within the .cairo/ folder
+cp target/release/starknet-sierra-compile ~/cairo_venv/lib/python3.9/site-packages/starkware/starknet/compiler/v1/bin/starknet-sierra-compile
+----
+____
+
+____
+NOTE: Once you have replaced the library, you can run the starknet declare command again and you should get a class hash as the output.
+____
+
+For example, a class hash that was deployed a while ago can be found on StarkScan https://testnet.starkscan.co/class/0x03ae6bb224f12a8818c3ad59cc248968e8ee3a3081ff805610141695381a28f7[here].
+
+Using the above generated class hash, deploy the contract:
+
+[,Bash]
+----
+starknet deploy --class_hash 0x03ae6bb224f12a8818c3ad59cc248968e8ee3a3081ff805610141695381a28f7 --account test_v11
+----
+
+____
+NOTE: P.S: If you run into any fee related issues, please add the flag  --max_fee 100000000000000000 to your CLI commands.
+____
+
+Monitor the deploy transaction. Once it has passed "pending", proceed.
+
+Wohooo! If all goes well, you have just deployed your first ever Cairo 1.0 contract on the StarkNet testnet! Congratulations.
+
++++<strong>+++Step 4: Interact with your contract+++</strong>+++
+
+If you quickly browse through the above contract(hello_starknet.cairo), you can see, the contract has a simple function: Say_Hello which we are going to learn to invoke in this step.
+
+[,Bash]
+----
+
+#Function that we will be invoking
+#[external]
+        fn Say_Hello(message:felt252) {
+            letcaller=get_caller_address();
+            Hello(caller, message);
+        }
+----
+
+The syntax to invoke a function in your contract is:
+
+[,Bash]
+----
+starknet invoke --function <name of the function> --address <address of the deployed contract> --max_fee 100000000000000000 --account <account_name>
+
+#Invoking our Say_Hello function
+starknet invoke --function Say_Hello --address 0x00e0309ca73c5097533ba200e08426b7efb3ee6b79ab6956bd4bb684d3385556 --input 152 --max_fee 100000000000000000  --account test_v11
+----
```

Comment:
same comment about casing for function names

Reply:
Fixed.